java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -292,6 +293,26 @@ void testGetNumberOfColumns() {
     }
   }
 
+  @Test
+  void testReadJSON() {
+    Schema schema = Schema.builder()
+        .column(DType.STRING, "A")
+        .column(DType.INT32, "B")
```

Comment:
How do "A" and "B" line up with "name" and "age" in the file? This feels really confusing because Spark goes off of the actual key "name" and "age". I would expect "A" and "B" to be null after reading.

Reply:
Done.

Reply:
Still a little confused.  Did the "A" and "B" work at all?

Reply:
No, specifying "A" "B" will not work. The first WIP commit can work is because column names were not passed to lib cudf.

Sorry for causing the confusion.